test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -446,9 +446,6 @@ func runUpgradeSuite(
 	upgradeType upgrades.UpgradeType,
 	upgradeFunc func(),
 ) {
```

Comment:
the invocation from here is not identical to what is being removed:
https://github.com/kubernetes/kubernetes/blob/833e8dc10bdce43fc8229c39639a7f7364703874/test/e2e/lifecycle/cluster_upgrade.go#L196-L208

Reply:
Ooo good catch! Going to remove `test/e2e/lifecycle` and just do `test/e2e/lifecycle/bootstrap` in this diff. I'll look into the differences here later.